Dockerfile.ui

```diff
@@ -29,6 +23,13 @@ COPY cvat-core/ /tmp/cvat-core/
 COPY cvat-canvas3d/ /tmp/cvat-canvas3d/
 COPY cvat-canvas/ /tmp/cvat-canvas/
 COPY cvat-ui/ /tmp/cvat-ui/
+
+ARG WA_PAGE_VIEW_HIT
```

Comment:
I think any mentioning of WA_PAGE_VIEW_HIT may be removed as we dropped google web analytics long time ago 

@azhavoro I am correct?

Reply:
FWIW, this variable is still used in the code.

Reply:
It is used in .ts code that also needs to be removed

Reply:
This code will work if `WA_PAGE_VIEW_HIT`, but it is not defined anywhere in saas repo as I can see

Reply:
Okay, I will send one more PR to solve it